.github/workflows/test.yml

```diff
@@ -26,18 +26,10 @@ jobs:
       fail-fast: false
       matrix:
         config:
-          # On Windows,
-          #
-          # * for R >= 4.2, both the MSVC toolchain and the GNU toolchain should
-          #   work. Since our main support is on MSVC, we mainly test MSVC here.
-          #   Also, at least one GNU case should be included so that we can
-          #   detect when something gets broken. 
-          # * for R < 4.2, the MSVC toolchain must be used to support
-          #   cross-compilation for the 32-bit. 
-          - {os: windows-latest, r: 'release', rust-version: 'stable-msvc', rtools-version: '42'}
-          - {os: windows-latest, r: 'devel',   rust-version: 'stable-msvc'}
-          - {os: windows-latest, r: 'release', rust-version: 'stable-gnu',  rtools-version: '42'}
-          - {os: windows-latest, r: '4.1',     rust-version: 'stable-msvc'}  # TODO: Remove this runner when we drop the support for R < 4.2
+          - {os: windows-latest, r: 'release', rust-version: 'stable-msvc', rtools-version: '43'}
+          - {os: windows-latest, r: 'devel',   rust-version: 'stable-msvc', rtools-version: '43'}
+          - {os: windows-latest, r: 'release', rust-version: 'stable-gnu',  rtools-version: '43'}
```

Comment:
What's the point in supporting `gnu` now? I dropped it from `{rextendr}` CI.

Reply:
I don't remember the reason why we included it, so it should be fine.

Reply:
Can we keep one gnu test for health?

Reply:
We used it because we made a switch from `gnu` to `msvc` and kept it for 'compatibility', though all of our workflows were using `msvc`